synapse/handlers/e2e_room_keys.py

```diff
@@ -97,7 +97,7 @@ async def get_room_keys(
                 user_id, version, room_id, session_id
             )
 
-            log_kv(results)
+            log_kv(cast(JsonDict, results))
```

Comment:
Bit of a shame we need the noise here. We could maybe change our `log_kv` wrapper to accept a `Mapping[str, Any]` aka `JsonMapping` instead of `JsonDict`... but it's probably not worth it!

(Fine as it is, I'm just lamenting in passing)